src/components/shared/Map/MapPointer.tsx

```diff
@@ -4,35 +4,51 @@ import { Circle, Marker, Popup } from 'react-leaflet';
 
 import { type SensorData } from '@/types/models/SensorData';
 
+import { COLORS, LEVELS } from '../../../constants/index';
```

Comment:
it was not working like this, but ok i changed it

Reply:
Well, it should be working. Maybe you haven't reloaded TypeScript server and it was displaying a problem with this path. One more thing, you don't have to use `index` in path. If you specify `@/constants` it will automatically import the `index.ts` files.